lib/bump/cli/commands/resolve.rb

```diff
@@ -0,0 +1,31 @@
+module Bump
+  class CLI
+    module Commands
+      class Resolve < Base
+        desc "Resolve a specification external references"
+        argument :file, required: true, desc: "Path or URL to your API documentation file. OpenAPI (2.0 to 3.0.2) and AsyncAPI (2.0) specifications are currently supported."
+        argument :destination, required: false, desc: "Destination file if not the same."
+        option :overwrite, type: :boolean, desc: "Overwrite the destination file if it already exists."
+
+        def call(file:, destination: nil, **options)
+          with_errors_rescued do
+            definition = Definition.new(file, import_external_references: true)
+            destination = destination.nil? ? file : destination
+            check_destination!(destination, options)
+
+            definition.prepare
+            definition.write(to: destination)
+
+            puts "External references have been resolved in #{destination}"
```

Comment:
What happens if `destination.nil?` ?

Maybe I missed something, but
I don't see how we avoid `definition.prepare`, `definition.write` or `puts "External references have been resolved in #{nil}"`

Reply:
Well, it cannot be nil, as it's always initialized here: https://github.com/bump-sh/bump-cli/pull/25/files#diff-6020f4356b1160073d2d70a41652b9ac8be84b15e8343182b030e8018cbfb7f5R13

Reply:
Oh Ok,

It looks very clear now that I read it again... And I don't even understand why I was confuse about.
Indeed `destination` is either `file`, either `Definition.new(file, import_external_references: true)`

My bad ... LGTM !